diff --git a/node_modules/antd/es/modal/Modal.js b/node_modules/antd/es/modal/Modal.js
index 2cea941..0321b9c 100644
--- a/node_modules/antd/es/modal/Modal.js
+++ b/node_modules/antd/es/modal/Modal.js
@@ -24,27 +24,8 @@ import Button from '../button';
 import { convertLegacyProps } from '../button/button';
 import LocaleReceiver from '../locale-provider/LocaleReceiver';
 import { ConfigContext } from '../config-provider';
-var mousePosition;
 export var destroyFns = []; // ref: https://github.com/ant-design/ant-design/issues/15795
 
-var getClickPosition = function getClickPosition(e) {
-  mousePosition = {
-    x: e.pageX,
-    y: e.pageY
-  }; // 100ms 内发生过点击事件，则从点击位置动画展示
-  // 否则直接 zoom 展示
-  // 这样可以兼容非点击方式展开
-
-  setTimeout(function () {
-    mousePosition = null;
-  }, 100);
-}; // 只有点击事件支持从鼠标位置动画展开
-
-
-if (typeof window !== 'undefined' && window.document && window.document.documentElement) {
-  document.documentElement.addEventListener('click', getClickPosition, true);
-}
-
 var Modal = function Modal(props) {
   var _classNames;
 
@@ -110,7 +91,6 @@ var Modal = function Modal(props) {
     wrapClassName: wrapClassNameExtended,
     footer: footer === undefined ? defaultFooter : footer,
     visible: visible,
-    mousePosition: mousePosition,
     onClose: handleCancel,
     closeIcon: closeIconToRender,
     focusTriggerAfterClose: focusTriggerAfterClose
