diff --git a/node_modules/@nrwl/node/src/utils/normalize.js b/node_modules/@nrwl/node/src/utils/normalize.js
index f6c5f0e..f2e79ac 100644
--- a/node_modules/@nrwl/node/src/utils/normalize.js
+++ b/node_modules/@nrwl/node/src/utils/normalize.js
@@ -21,9 +21,6 @@ function normalizeAssets(assets, root, sourceRoot) {
         if (typeof asset === 'string') {
             const resolvedAssetPath = path_1.resolve(root, asset);
             const resolvedSourceRoot = path_1.resolve(root, sourceRoot);
-            if (!resolvedAssetPath.startsWith(resolvedSourceRoot)) {
-                throw new Error(`The ${resolvedAssetPath} asset path must start with the project source root: ${sourceRoot}`);
-            }
             const isDirectory = fs_1.statSync(resolvedAssetPath).isDirectory();
             const input = isDirectory
                 ? resolvedAssetPath
